js/search.js

```diff
@@ -36,7 +48,30 @@ function(ytData, playlist, template, $) {
     }
 
     var displaySearchResults = function(){
-        $("#search_res").empty()
+        $("#search_res").empty();
+        $("#search_res").append("<li><h6 class='ti'>Playlist</h6></li>");
+        if(searchPlaylist===null){
+            $("#search_res").append("<p>Nothing found</p>");
+        }else{
+            for(song in searchPlaylist){
+                var vidrow=template.getVidRow(searchPlaylist[song]);
+                $(vidrow).addClass("item-search");
+                $(vidrow).on("click",function(){
+                    //console.log('clicked '+this.id);
+                    var hashId="#playlist > #"+this.id;
+                    $("#playlist").animate({
+                            scrollTop : $(hashId).offset().top
+                        }, 2000);
+                    $(hashId).css({'background-color':'#56BD81'});
+                    setTimeout(function(){
+                        $(hashId).css({'background-color':'#fff'});
+                    },3000);
+                });
+                $(vidrow).find("#action").css({'display': 'none'});
+                vidrow.appendTo("#search_res");
+            }
+        }
+        $("#search_res").append("<li><h6 class='ti'>Youtube</h6></li>");
```

Comment:
Because these titles are static, they can be initially declared using `$('<li><h6 class='ti'>Youtube</h6></li>')`, and added here, similar to what is done [here](https://github.com/udiboy1209/fast_playlist/blob/master/js/template.js#L2). This kind of declaration is faster because it doesn't have to inflate DOM elements from HTML every time.